src/cluster/k8sClientHandler.go

```diff
@@ -557,6 +555,97 @@ func GetStatefulSetsFromK8sClient() []types.Deployment {
 					Name:      sts.Name,
 					Namespace: sts.Namespace,
 					Labels:    strings.Join(labels, ","),
+					Type:      types.K8sStatefulSetsType,
+				})
+			}
+		}
+	}
+	return results
+}
+
+// ================= //
+// == DaemonSets == //
+// ================= //
+
+func GetDaemonSetsFromK8sClient() []types.Deployment {
```

Comment:
Can we create a single function and then pass the k8s objects as argument? like GetObjectSetsFromK8sClient(object), then pass daemonset/deployment etc. as a value to it. It will greatly reduce the code size

Reply:
Addressed @Ankurk99